src/SFA.DAS.LoginService.Web/Infrastructure/CustomPasswordValidator.cs

```diff
@@ -25,7 +25,7 @@ public async Task<IdentityResult> ValidateAsync(UserManager<TUser> manager, TUse
             if (string.IsNullOrWhiteSpace(password) ||
                 password.Length < 8 || 
                 password.All(char.IsDigit) || 
-                password.All(char.IsLetter))
+                password.All(char.IsLetter) || !PasswordContainsBothLettersAndDigits(password))
```

Comment:
On an incredibly nit-picking level, could we make all the conditions line up?  It's a bit weird to see some of them on individual lines and two on the same line

Reply:
ok